index.js

```diff
@@ -0,0 +1,38 @@
+const yaml = require('js-yaml');
+
+module.exports = robot => {
+    robot.on('pull_request.opened', async context => {
+        let config;
+        const files = await context.github.pullRequests.getFiles(context.issue());
+        const docs = files.data.find(function (file) {
+            if (file.filename === 'README.md' || file.filename.includes('docs/')) {
```

Comment:
> (does anyone do that?)

Yepp, the lisp + emacs folk do. They write all their READMEs in OrgMode (`README.org`).